meta-balena-common/recipes-containers/hostapp-update/files/hostapp-update

```diff
@@ -23,6 +23,14 @@ ERROR() {
 	fi
 }
 
+WARN() {
+	if command -v warn > /dev/null; then
+		warn "$@"
+	else
+		echo "$@"
+	fi
+}
+
```

Comment:
Why not use os-helpers-logging instead?

Reply:
This does just that, it uses the `warn` function from `os-helpers-logging`

Reply:
Right, why checking whether warn() is available then? On what scenario `os-helpers-logging` won't be in the hostOS?

Reply:
It should always be there, you are right. I'll just remove the safety check